etcdserver: add ConfChangeAddLearnerNode to the list of config changes

galal-hussein opened the pull request:
To fix a panic that happens when force-new-cluster flag is passed to
etcd node if the cluster had learner nodes added from before

Fixes #12285

jingyih reviewed (commented):
LGTM
Just a nit: could you also update line 617: `s/two/three`

jingyih commented:
We will need to backport this to 3.4 branch.

galal-hussein commented:
@jingyih thanks very much, fixed the nit and added the backport PR https://github.com/etcd-io/etcd/compare/release-3.4...galal-hussein:fix_panic_34?expand=1

spzala reviewed (commented):
lgtm Thanks @galal-hussein

jingyih commented:
For the reference, this PR is backported to release-3.4 via #12299.
